commit 33e0c5714b4f9d45a34618dadc55f6b248aeef74 (refs/changes/35/4935/4)
Author: John Levon <john.levon@joyent.com>
Date:   2018-10-11T12:52:24+00:00 (12 months ago)
    
    OS-7297 libresolv2_joy needs fixes for newer GCC
    Reviewed by: Mike Gerdts <mike.gerdts@joyent.com>
    Reviewed by: Dan McDonald <danmcd@joyent.com>
    Approved by: Dan McDonald <danmcd@joyent.com>

diff --git a/exception_lists/cstyle b/exception_lists/cstyle
index 2b6b49a0a2..af001c1cc6 100644
--- a/exception_lists/cstyle
+++ b/exception_lists/cstyle
@@ -689,6 +689,7 @@ usr/src/lib/libdns_sd/common/dns_sd.h
 usr/src/lib/libdns_sd/java/common/JNISupport.c
 usr/src/lib/libgss/g_glue.c
 usr/src/lib/libresolv2/common
+usr/src/lib/libresolv2_joy/common
 usr/src/lib/librstp/common/base.h
 usr/src/lib/librstp/common/choose.h
 usr/src/lib/librstp/common/edge.c
diff --git a/exception_lists/wscheck b/exception_lists/wscheck
index 0d06b13802..7d0795bc7f 100644
--- a/exception_lists/wscheck
+++ b/exception_lists/wscheck
@@ -14,6 +14,8 @@ syntax: glob
 
 usr/src/data/ucode/amd/*
 usr/src/data/ucode/intel/*
+usr/src/lib/libresolv2/common/*
+usr/src/lib/libresolv2_joy/common/*
 
 # bhyve sources
 usr/src/cmd/bhyve/acpi.[ch]
diff --git a/usr/src/lib/libresolv2/common/nameser/ns_print.c b/usr/src/lib/libresolv2/common/nameser/ns_print.c
index a311051144..fc228e753f 100644
--- a/usr/src/lib/libresolv2/common/nameser/ns_print.c
+++ b/usr/src/lib/libresolv2/common/nameser/ns_print.c
@@ -1,6 +1,7 @@
 /*
  * Copyright (c) 2004 by Internet Systems Consortium, Inc. ("ISC")
  * Copyright (c) 1996-1999 by Internet Software Consortium.
+ * Copyright 2018 Joyent, Inc.
  *
  * Permission to use, copy, modify, and distribute this software for any
  * purpose with or without fee is hereby granted, provided that the above
@@ -894,8 +895,8 @@ ns_sprintrrf(const u_char *msg, size_t msglen,
 			if (len > 15)
 				T(addstr(" )", 2, &buf, &buflen));
 		}
+		break;
 	}
-	/* FALLTHROUGH */
 
 	case ns_t_ipseckey: {
 		int n;
@@ -986,8 +987,8 @@ ns_sprintrrf(const u_char *msg, size_t msglen,
 			if (len > 15)
 				T(addstr(" )", 2, &buf, &buflen));
 		}
+		break;
 	}
-	/* FALLTHROUGH */
 
 	case ns_t_hip: {
 		unsigned int i, hip_len, algorithm, key_len;
diff --git a/usr/src/lib/libresolv2_joy/common/bsd/ftruncate.c b/usr/src/lib/libresolv2_joy/common/bsd/ftruncate.c
index 5ac4ebac9b..744d205ce7 100644
--- a/usr/src/lib/libresolv2_joy/common/bsd/ftruncate.c
+++ b/usr/src/lib/libresolv2_joy/common/bsd/ftruncate.c
@@ -1,7 +1,3 @@
-#ifndef LINT
-static const char rcsid[] = "$Id: ftruncate.c,v 1.3 2005/04/27 18:16:45 sra Exp $";
-#endif
-
 /*! \file
  * \brief
  * ftruncate - set file size, BSD Style
diff --git a/usr/src/lib/libresolv2_joy/common/bsd/gettimeofday.c b/usr/src/lib/libresolv2_joy/common/bsd/gettimeofday.c
index 2926a3575e..f5ff8c54f5 100644
--- a/usr/src/lib/libresolv2_joy/common/bsd/gettimeofday.c
+++ b/usr/src/lib/libresolv2_joy/common/bsd/gettimeofday.c
@@ -1,7 +1,3 @@
-#ifndef LINT
-static const char rcsid[] = "$Id: gettimeofday.c,v 1.4 2005/04/27 04:56:11 sra Exp $";
-#endif
-
 #include "port_before.h"
 #include <stdio.h>
 #include <syslog.h>
diff --git a/usr/src/lib/libresolv2_joy/common/bsd/mktemp.c b/usr/src/lib/libresolv2_joy/common/bsd/mktemp.c
index 001b24b58f..2d41b9b0ee 100644
--- a/usr/src/lib/libresolv2_joy/common/bsd/mktemp.c
+++ b/usr/src/lib/libresolv2_joy/common/bsd/mktemp.c
@@ -1,8 +1,3 @@
-#if defined(LIBC_SCCS) && !defined(lint)
-static const char sccsid[] = "@(#)mktemp.c	8.1 (Berkeley) 6/4/93";
-static const char rcsid[] = "$Id: mktemp.c,v 1.2 2005/04/27 04:56:11 sra Exp $";
-#endif /* LIBC_SCCS and not lint */
-
 /*
  * Copyright (c) 1987, 1993
  *    The Regents of the University of California.  All rights reserved.
diff --git a/usr/src/lib/libresolv2_joy/common/bsd/putenv.c b/usr/src/lib/libresolv2_joy/common/bsd/putenv.c
index 2dcbc57e6c..de1b070de6 100644
--- a/usr/src/lib/libresolv2_joy/common/bsd/putenv.c
+++ b/usr/src/lib/libresolv2_joy/common/bsd/putenv.c
@@ -1,7 +1,3 @@
-#ifndef LINT
-static const char rcsid[] = "$Id: putenv.c,v 1.2 2005/04/27 04:56:11 sra Exp $";
-#endif
-
 #include "port_before.h"
 #include "port_after.h"
 
diff --git a/usr/src/lib/libresolv2_joy/common/bsd/readv.c b/usr/src/lib/libresolv2_joy/common/bsd/readv.c
index 5fa691a92f..0a8ade0ec1 100644
--- a/usr/src/lib/libresolv2_joy/common/bsd/readv.c
+++ b/usr/src/lib/libresolv2_joy/common/bsd/readv.c
@@ -1,7 +1,3 @@
-#ifndef LINT
-static const char rcsid[] = "$Id: readv.c,v 1.2 2005/04/27 04:56:11 sra Exp $";
-#endif
-
 #include "port_before.h"
 
 #include <sys/types.h>
diff --git a/usr/src/lib/libresolv2_joy/common/bsd/setenv.c b/usr/src/lib/libresolv2_joy/common/bsd/setenv.c
index baf00f6ff2..f9fdbb7be4 100644
--- a/usr/src/lib/libresolv2_joy/common/bsd/setenv.c
+++ b/usr/src/lib/libresolv2_joy/common/bsd/setenv.c
@@ -1,8 +1,3 @@
-#if defined(LIBC_SCCS) && !defined(lint)
-static const char sccsid[] = "@(#)setenv.c	8.1 (Berkeley) 6/4/93";
-static const char rcsid[] = "$Id: setenv.c,v 1.2 2005/04/27 04:56:11 sra Exp $";
-#endif /* LIBC_SCCS and not lint */
-
 /*
  * Copyright (c) 1987, 1993
  *	The Regents of the University of California.  All rights reserved.
diff --git a/usr/src/lib/libresolv2_joy/common/bsd/setitimer.c b/usr/src/lib/libresolv2_joy/common/bsd/setitimer.c
index 67881d7ca8..d336ac80d9 100644
--- a/usr/src/lib/libresolv2_joy/common/bsd/setitimer.c
+++ b/usr/src/lib/libresolv2_joy/common/bsd/setitimer.c
@@ -1,7 +1,3 @@
-#ifndef LINT
-static const char rcsid[] = "$Id: setitimer.c,v 1.2 2005/04/27 04:56:12 sra Exp $";
-#endif
-
 #include "port_before.h"
 
 #include <sys/time.h>
diff --git a/usr/src/lib/libresolv2_joy/common/bsd/strcasecmp.c b/usr/src/lib/libresolv2_joy/common/bsd/strcasecmp.c
index 0c9f0dccf0..adf76ae483 100644
--- a/usr/src/lib/libresolv2_joy/common/bsd/strcasecmp.c
+++ b/usr/src/lib/libresolv2_joy/common/bsd/strcasecmp.c
@@ -1,8 +1,3 @@
-#if defined(LIBC_SCCS) && !defined(lint)
-static const char sccsid[] = "@(#)strcasecmp.c	8.1 (Berkeley) 6/4/93";
-static const char rcsid[] = "$Id: strcasecmp.c,v 1.2 2005/04/27 04:56:12 sra Exp $";
-#endif /* LIBC_SCCS and not lint */
-
 /*
  * Copyright (c) 1987, 1993
  *    The Regents of the University of California.  All rights reserved.
diff --git a/usr/src/lib/libresolv2_joy/common/bsd/strerror.c b/usr/src/lib/libresolv2_joy/common/bsd/strerror.c
index 5973e63d5b..107215c798 100644
--- a/usr/src/lib/libresolv2_joy/common/bsd/strerror.c
+++ b/usr/src/lib/libresolv2_joy/common/bsd/strerror.c
@@ -1,8 +1,3 @@
-#if defined(LIBC_SCCS) && !defined(lint)
-static const char sccsid[] = "@(#)strerror.c	8.1 (Berkeley) 6/4/93";
-static const char rcsid[] = "$Id: strerror.c,v 1.6 2008/02/18 03:49:08 marka Exp $";
-#endif /* LIBC_SCCS and not lint */
-
 /*
  * Copyright (c) 1988, 1993
  *    The Regents of the University of California.  All rights reserved.
diff --git a/usr/src/lib/libresolv2_joy/common/bsd/strpbrk.c b/usr/src/lib/libresolv2_joy/common/bsd/strpbrk.c
index 4c12d88e1c..01e08dd5f4 100644
--- a/usr/src/lib/libresolv2_joy/common/bsd/strpbrk.c
+++ b/usr/src/lib/libresolv2_joy/common/bsd/strpbrk.c
@@ -1,8 +1,3 @@
-#if defined(LIBC_SCCS) && !defined(lint)
-static const char sccsid[] = "@(#)strpbrk.c	8.1 (Berkeley) 6/4/93";
-static const char rcsid[] = "$Id: strpbrk.c,v 1.2 2005/04/27 04:56:12 sra Exp $";
-#endif /* LIBC_SCCS and not lint */
-
 /*
  * Copyright (c) 1985, 1993
  *    The Regents of the University of California.  All rights reserved.
diff --git a/usr/src/lib/libresolv2_joy/common/bsd/strsep.c b/usr/src/lib/libresolv2_joy/common/bsd/strsep.c
index c7969f0028..0470733543 100644
--- a/usr/src/lib/libresolv2_joy/common/bsd/strsep.c
+++ b/usr/src/lib/libresolv2_joy/common/bsd/strsep.c
@@ -1,8 +1,3 @@
-#if defined(LIBC_SCCS) && !defined(lint)
-static const char sccsid[] = "strsep.c	8.1 (Berkeley) 6/4/93";
-static const char rcsid[] = "$Id: strsep.c,v 1.2 2005/04/27 04:56:12 sra Exp $";
-#endif /* LIBC_SCCS and not lint */
-
 /*
  * Copyright (c) 1990, 1993
  *	The Regents of the University of California.  All rights reserved.
diff --git a/usr/src/lib/libresolv2_joy/common/bsd/strtoul.c b/usr/src/lib/libresolv2_joy/common/bsd/strtoul.c
index b37ff72729..bc199708b7 100644
--- a/usr/src/lib/libresolv2_joy/common/bsd/strtoul.c
+++ b/usr/src/lib/libresolv2_joy/common/bsd/strtoul.c
@@ -1,8 +1,3 @@
-#if defined(LIBC_SCCS) && !defined(lint)
-static const char sccsid[] = "@(#)strtoul.c	8.1 (Berkeley) 6/4/93";
-static const char rcsid[] = "$Id: strtoul.c,v 1.4 2008/02/18 03:49:08 marka Exp $";
-#endif /* LIBC_SCCS and not lint */
-
 /*
  * Copyright (c) 1990, 1993
  *    The Regents of the University of California.  All rights reserved.
diff --git a/usr/src/lib/libresolv2_joy/common/bsd/writev.c b/usr/src/lib/libresolv2_joy/common/bsd/writev.c
index 65baa71cfc..086f93266b 100644
--- a/usr/src/lib/libresolv2_joy/common/bsd/writev.c
+++ b/usr/src/lib/libresolv2_joy/common/bsd/writev.c
@@ -1,7 +1,3 @@
-#ifndef LINT
-static const char rcsid[] = "$Id: writev.c,v 1.3 2005/04/27 04:56:13 sra Exp $";
-#endif
-
 #include "port_before.h"
 
 #include <sys/types.h>
diff --git a/usr/src/lib/libresolv2_joy/common/dst/dst_api.c b/usr/src/lib/libresolv2_joy/common/dst/dst_api.c
index a3e48077ad..7215772ce2 100644
--- a/usr/src/lib/libresolv2_joy/common/dst/dst_api.c
+++ b/usr/src/lib/libresolv2_joy/common/dst/dst_api.c
@@ -1,7 +1,3 @@
-#ifndef LINT
-static const char rcsid[] = "$Header: /proj/cvs/prod/libbind/dst/dst_api.c,v 1.17 2007/09/24 17:18:25 each Exp $";
-#endif
-
 /*
  * Portions Copyright (c) 1995-1998 by Trusted Information Systems, Inc.
  *
diff --git a/usr/src/lib/libresolv2_joy/common/dst/hmac_link.c b/usr/src/lib/libresolv2_joy/common/dst/hmac_link.c
index 23925e4269..f005784e7b 100644
--- a/usr/src/lib/libresolv2_joy/common/dst/hmac_link.c
+++ b/usr/src/lib/libresolv2_joy/common/dst/hmac_link.c
@@ -5,9 +5,6 @@
 
 
 #ifdef HMAC_MD5
-#ifndef LINT
-static const char rcsid[] = "$Header: /proj/cvs/prod/libbind/dst/hmac_link.c,v 1.8 2007/09/24 17:18:25 each Exp $";
-#endif
 /*
  * Portions Copyright (c) 1995-1998 by Trusted Information Systems, Inc.
  *
diff --git a/usr/src/lib/libresolv2_joy/common/dst/support.c b/usr/src/lib/libresolv2_joy/common/dst/support.c
index 8f827667d6..cc975284b6 100644
--- a/usr/src/lib/libresolv2_joy/common/dst/support.c
+++ b/usr/src/lib/libresolv2_joy/common/dst/support.c
@@ -1,6 +1,3 @@
-static const char rcsid[] = "$Header: /proj/cvs/prod/libbind/dst/support.c,v 1.6 2005/10/11 00:10:13 marka Exp $";
-
-
 /*
  * Portions Copyright (c) 1995-1998 by Trusted Information Systems, Inc.
  *
diff --git a/usr/src/lib/libresolv2_joy/common/irs/dns.c b/usr/src/lib/libresolv2_joy/common/irs/dns.c
index 7c50320ae7..119308074e 100644
--- a/usr/src/lib/libresolv2_joy/common/irs/dns.c
+++ b/usr/src/lib/libresolv2_joy/common/irs/dns.c
@@ -15,10 +15,6 @@
  * OF OR IN CONNECTION WITH THE USE OR PERFORMANCE OF THIS SOFTWARE.
  */
 
-#if defined(LIBC_SCCS) && !defined(lint)
-static const char rcsid[] = "$Id: dns.c,v 1.5 2006/03/09 23:57:56 marka Exp $";
-#endif
-
 /*! \file
  * \brief
  * dns.c --- this is the top-level accessor function for the dns
diff --git a/usr/src/lib/libresolv2_joy/common/irs/dns_ho.c b/usr/src/lib/libresolv2_joy/common/irs/dns_ho.c
index 67812717fb..b47e26f21a 100644
--- a/usr/src/lib/libresolv2_joy/common/irs/dns_ho.c
+++ b/usr/src/lib/libresolv2_joy/common/irs/dns_ho.c
@@ -51,10 +51,6 @@
 /* from gethostnamadr.c	8.1 (Berkeley) 6/4/93 */
 /* BIND Id: gethnamaddr.c,v 8.15 1996/05/22 04:56:30 vixie Exp $ */
 
-#if defined(LIBC_SCCS) && !defined(lint)
-static const char rcsid[] = "$Id: dns_ho.c,v 1.23 2008/11/14 02:36:51 marka Exp $";
-#endif /* LIBC_SCCS and not lint */
-
 /* Imports. */
 
 #include "port_before.h"
diff --git a/usr/src/lib/libresolv2_joy/common/irs/dns_nw.c b/usr/src/lib/libresolv2_joy/common/irs/dns_nw.c
index e9acfd39c7..9d9dbf9cd2 100644
--- a/usr/src/lib/libresolv2_joy/common/irs/dns_nw.c
+++ b/usr/src/lib/libresolv2_joy/common/irs/dns_nw.c
@@ -15,10 +15,6 @@
  * OF OR IN CONNECTION WITH THE USE OR PERFORMANCE OF THIS SOFTWARE.
  */
 
-#if defined(LIBC_SCCS) && !defined(lint)
-static const char rcsid[] = "$Id: dns_nw.c,v 1.12 2005/04/27 04:56:22 sra Exp $";
-#endif /* LIBC_SCCS and not lint */
-
 /* Imports. */
 
 #include "port_before.h"
diff --git a/usr/src/lib/libresolv2_joy/common/irs/dns_pr.c b/usr/src/lib/libresolv2_joy/common/irs/dns_pr.c
index c281be432c..0fb533e656 100644
--- a/usr/src/lib/libresolv2_joy/common/irs/dns_pr.c
+++ b/usr/src/lib/libresolv2_joy/common/irs/dns_pr.c
@@ -15,10 +15,6 @@
  * OF OR IN CONNECTION WITH THE USE OR PERFORMANCE OF THIS SOFTWARE.
  */
 
-#if defined(LIBC_SCCS) && !defined(lint)
-static const char rcsid[] = "$Id: dns_pr.c,v 1.5 2005/04/27 04:56:22 sra Exp $";
-#endif
-
 /* Imports */
 
 #include "port_before.h"
diff --git a/usr/src/lib/libresolv2_joy/common/irs/dns_sv.c b/usr/src/lib/libresolv2_joy/common/irs/dns_sv.c
index 1001dc1051..720c230595 100644
--- a/usr/src/lib/libresolv2_joy/common/irs/dns_sv.c
+++ b/usr/src/lib/libresolv2_joy/common/irs/dns_sv.c
@@ -15,10 +15,6 @@
  * OF OR IN CONNECTION WITH THE USE OR PERFORMANCE OF THIS SOFTWARE.
  */
 
-#if defined(LIBC_SCCS) && !defined(lint)
-static const char rcsid[] = "$Id: dns_sv.c,v 1.5 2005/04/27 04:56:23 sra Exp $";
-#endif
-
 /* Imports */
 
 #include "port_before.h"
diff --git a/usr/src/lib/libresolv2_joy/common/irs/gen.c b/usr/src/lib/libresolv2_joy/common/irs/gen.c
index 7da01f5b09..98203149e9 100644
--- a/usr/src/lib/libresolv2_joy/common/irs/gen.c
+++ b/usr/src/lib/libresolv2_joy/common/irs/gen.c
@@ -21,10 +21,6 @@
  * OF OR IN CONNECTION WITH THE USE OR PERFORMANCE OF THIS SOFTWARE.
  */
 
-#if !defined(LINT) && !defined(CODECENTER)
-static const char rcsid[] = "$Id: gen.c,v 1.7 2005/04/27 04:56:23 sra Exp $";
-#endif
-
 /*! \file
  * \brief
  * this is the top level dispatcher
diff --git a/usr/src/lib/libresolv2_joy/common/irs/gen_ho.c b/usr/src/lib/libresolv2_joy/common/irs/gen_ho.c
index 8e41d7d610..f8eddd4c0e 100644
--- a/usr/src/lib/libresolv2_joy/common/irs/gen_ho.c
+++ b/usr/src/lib/libresolv2_joy/common/irs/gen_ho.c
@@ -15,10 +15,6 @@
  * OF OR IN CONNECTION WITH THE USE OR PERFORMANCE OF THIS SOFTWARE.
  */
 
-#if defined(LIBC_SCCS) && !defined(lint)
-static const char rcsid[] = "$Id: gen_ho.c,v 1.5 2006/03/09 23:57:56 marka Exp $";
-#endif /* LIBC_SCCS and not lint */
-
 /* Imports */
 
 #include "port_before.h"
diff --git a/usr/src/lib/libresolv2_joy/common/irs/gen_ng.c b/usr/src/lib/libresolv2_joy/common/irs/gen_ng.c
index e3c874ee68..05c7d4dfb7 100644
--- a/usr/src/lib/libresolv2_joy/common/irs/gen_ng.c
+++ b/usr/src/lib/libresolv2_joy/common/irs/gen_ng.c
@@ -15,10 +15,6 @@
  * OF OR IN CONNECTION WITH THE USE OR PERFORMANCE OF THIS SOFTWARE.
  */
 
-#if !defined(LINT) && !defined(CODECENTER)
-static const char rcsid[] = "$Id: gen_ng.c,v 1.3 2005/04/27 04:56:23 sra Exp $";
-#endif
-
 /* Imports */
 
 #include "port_before.h"
diff --git a/usr/src/lib/libresolv2_joy/common/irs/gen_nw.c b/usr/src/lib/libresolv2_joy/common/irs/gen_nw.c
index 12bd0e0f6d..a84a83af98 100644
--- a/usr/src/lib/libresolv2_joy/common/irs/gen_nw.c
+++ b/usr/src/lib/libresolv2_joy/common/irs/gen_nw.c
@@ -15,10 +15,6 @@
  * OF OR IN CONNECTION WITH THE USE OR PERFORMANCE OF THIS SOFTWARE.
  */
 
-#if !defined(LINT) && !defined(CODECENTER)
-static const char rcsid[] = "$Id: gen_nw.c,v 1.4 2005/04/27 04:56:23 sra Exp $";
-#endif
-
 /* Imports */
 
 #include "port_before.h"
diff --git a/usr/src/lib/libresolv2_joy/common/irs/gen_pr.c b/usr/src/lib/libresolv2_joy/common/irs/gen_pr.c
index 9fd32c4dd9..0349debe44 100644
--- a/usr/src/lib/libresolv2_joy/common/irs/gen_pr.c
+++ b/usr/src/lib/libresolv2_joy/common/irs/gen_pr.c
@@ -15,10 +15,6 @@
  * OF OR IN CONNECTION WITH THE USE OR PERFORMANCE OF THIS SOFTWARE.
  */
 
-#if !defined(LINT) && !defined(CODECENTER)
-static const char rcsid[] = "$Id: gen_pr.c,v 1.3 2005/04/27 04:56:24 sra Exp $";
-#endif
-
 /* Imports */
 
 #include "port_before.h"
diff --git a/usr/src/lib/libresolv2_joy/common/irs/gen_sv.c b/usr/src/lib/libresolv2_joy/common/irs/gen_sv.c
index 93b70e57ec..a6f29d2a0f 100644
--- a/usr/src/lib/libresolv2_joy/common/irs/gen_sv.c
+++ b/usr/src/lib/libresolv2_joy/common/irs/gen_sv.c
@@ -15,10 +15,6 @@
  * OF OR IN CONNECTION WITH THE USE OR PERFORMANCE OF THIS SOFTWARE.
  */
 
-#if !defined(LINT) && !defined(CODECENTER)
-static const char rcsid[] = "$Id: gen_sv.c,v 1.3 2005/04/27 04:56:24 sra Exp $";
-#endif
-
 /* Imports */
 
 #include "port_before.h"
diff --git a/usr/src/lib/libresolv2_joy/common/irs/gethostent.c b/usr/src/lib/libresolv2_joy/common/irs/gethostent.c
index c4751c2c80..77c3e0a890 100644
--- a/usr/src/lib/libresolv2_joy/common/irs/gethostent.c
+++ b/usr/src/lib/libresolv2_joy/common/irs/gethostent.c
@@ -21,10 +21,6 @@
  * OF OR IN CONNECTION WITH THE USE OR PERFORMANCE OF THIS SOFTWARE.
  */
 
-#if !defined(LINT) && !defined(CODECENTER)
-static const char rcsid[] = "$Id: gethostent.c,v 1.8 2006/01/10 05:06:00 marka Exp $";
-#endif
-
 /* Imports */
 
 #include "port_before.h"
diff --git a/usr/src/lib/libresolv2_joy/common/irs/gethostent_r.c b/usr/src/lib/libresolv2_joy/common/irs/gethostent_r.c
index 1971677c5d..50e03f5773 100644
--- a/usr/src/lib/libresolv2_joy/common/irs/gethostent_r.c
+++ b/usr/src/lib/libresolv2_joy/common/irs/gethostent_r.c
@@ -15,10 +15,6 @@
  * OF OR IN CONNECTION WITH THE USE OR PERFORMANCE OF THIS SOFTWARE.
  */
 
-#if defined(LIBC_SCCS) && !defined(lint)
-static const char rcsid[] = "$Id: gethostent_r.c,v 1.9 2005/09/03 12:41:37 marka Exp $";
-#endif /* LIBC_SCCS and not lint */
-
 #include <port_before.h>
 #if !defined(_REENTRANT) || !defined(DO_PTHREADS)
 	static int gethostent_r_not_required = 0;
diff --git a/usr/src/lib/libresolv2_joy/common/irs/getnetent.c b/usr/src/lib/libresolv2_joy/common/irs/getnetent.c
index 0d00699e81..b3ba37547b 100644
--- a/usr/src/lib/libresolv2_joy/common/irs/getnetent.c
+++ b/usr/src/lib/libresolv2_joy/common/irs/getnetent.c
@@ -15,10 +15,6 @@
  * OF OR IN CONNECTION WITH THE USE OR PERFORMANCE OF THIS SOFTWARE.
  */
 
-#if !defined(LINT) && !defined(CODECENTER)
-static const char rcsid[] = "$Id: getnetent.c,v 1.7 2005/04/27 04:56:25 sra Exp $";
-#endif
-
 /* Imports */
 
 #include "port_before.h"
diff --git a/usr/src/lib/libresolv2_joy/common/irs/getnetent_r.c b/usr/src/lib/libresolv2_joy/common/irs/getnetent_r.c
index 9fb52bc394..df6831ff12 100644
--- a/usr/src/lib/libresolv2_joy/common/irs/getnetent_r.c
+++ b/usr/src/lib/libresolv2_joy/common/irs/getnetent_r.c
@@ -15,10 +15,6 @@
  * OF OR IN CONNECTION WITH THE USE OR PERFORMANCE OF THIS SOFTWARE.
  */
 
-#if defined(LIBC_SCCS) && !defined(lint)
-static const char rcsid[] = "$Id: getnetent_r.c,v 1.6 2005/09/03 12:41:38 marka Exp $";
-#endif /* LIBC_SCCS and not lint */
-
 #include <port_before.h>
 #if !defined(_REENTRANT) || !defined(DO_PTHREADS)
 	static int getnetent_r_not_required = 0;
diff --git a/usr/src/lib/libresolv2_joy/common/irs/getnetgrent.c b/usr/src/lib/libresolv2_joy/common/irs/getnetgrent.c
index 40b3e5a8ad..dbcd1da486 100644
--- a/usr/src/lib/libresolv2_joy/common/irs/getnetgrent.c
+++ b/usr/src/lib/libresolv2_joy/common/irs/getnetgrent.c
@@ -15,10 +15,6 @@
  * PERFORMANCE OF THIS SOFTWARE.
  */
 
-#if defined(LIBC_SCCS) && !defined(lint)
-static const char rcsid[] = "$Id: getnetgrent.c,v 1.6 2008/11/14 02:36:51 marka Exp $";
-#endif /* LIBC_SCCS and not lint */
-
 /* Imports */
 
 #include "port_before.h"
diff --git a/usr/src/lib/libresolv2_joy/common/irs/getnetgrent_r.c b/usr/src/lib/libresolv2_joy/common/irs/getnetgrent_r.c
index aa8810320d..2cc697910e 100644
--- a/usr/src/lib/libresolv2_joy/common/irs/getnetgrent_r.c
+++ b/usr/src/lib/libresolv2_joy/common/irs/getnetgrent_r.c
@@ -15,10 +15,6 @@
  * PERFORMANCE OF THIS SOFTWARE.
  */
 
-#if defined(LIBC_SCCS) && !defined(lint)
-static const char rcsid[] = "$Id: getnetgrent_r.c,v 1.14 2008/11/14 02:36:51 marka Exp $";
-#endif /* LIBC_SCCS and not lint */
-
 #include <port_before.h>
 #if !defined(_REENTRANT) || !defined(DO_PTHREADS)
 	static int getnetgrent_r_not_required = 0;
diff --git a/usr/src/lib/libresolv2_joy/common/irs/getprotoent.c b/usr/src/lib/libresolv2_joy/common/irs/getprotoent.c
index 32a1040916..e56ad4ab5d 100644
--- a/usr/src/lib/libresolv2_joy/common/irs/getprotoent.c
+++ b/usr/src/lib/libresolv2_joy/common/irs/getprotoent.c
@@ -15,10 +15,6 @@
  * OF OR IN CONNECTION WITH THE USE OR PERFORMANCE OF THIS SOFTWARE.
  */
 
-#if !defined(LINT) && !defined(CODECENTER)
-static const char rcsid[] = "$Id: getprotoent.c,v 1.4 2005/04/27 04:56:26 sra Exp $";
-#endif
-
 /* Imports */
 
 #include "port_before.h"
diff --git a/usr/src/lib/libresolv2_joy/common/irs/getprotoent_r.c b/usr/src/lib/libresolv2_joy/common/irs/getprotoent_r.c
index d5d9ae53b6..b9a2623bf1 100644
--- a/usr/src/lib/libresolv2_joy/common/irs/getprotoent_r.c
+++ b/usr/src/lib/libresolv2_joy/common/irs/getprotoent_r.c
@@ -15,10 +15,6 @@
  * OF OR IN CONNECTION WITH THE USE OR PERFORMANCE OF THIS SOFTWARE.
  */
 
-#if defined(LIBC_SCCS) && !defined(lint)
-static const char rcsid[] = "$Id: getprotoent_r.c,v 1.6 2006/08/01 01:14:16 marka Exp $";
-#endif /* LIBC_SCCS and not lint */
-
 #include <port_before.h>
 #if !defined(_REENTRANT) || !defined(DO_PTHREADS)
 	static int getprotoent_r_not_required = 0;
diff --git a/usr/src/lib/libresolv2_joy/common/irs/getservent.c b/usr/src/lib/libresolv2_joy/common/irs/getservent.c
index 31af67e659..3cc2422b98 100644
--- a/usr/src/lib/libresolv2_joy/common/irs/getservent.c
+++ b/usr/src/lib/libresolv2_joy/common/irs/getservent.c
@@ -15,10 +15,6 @@
  * OF OR IN CONNECTION WITH THE USE OR PERFORMANCE OF THIS SOFTWARE.
  */
 
-#if !defined(LINT) && !defined(CODECENTER)
-static const char rcsid[] = "$Id: getservent.c,v 1.4 2005/04/27 04:56:26 sra Exp $";
-#endif
-
 /* Imports */
 
 #include "port_before.h"
diff --git a/usr/src/lib/libresolv2_joy/common/irs/getservent_r.c b/usr/src/lib/libresolv2_joy/common/irs/getservent_r.c
index 42d1e46163..91cebcf480 100644
--- a/usr/src/lib/libresolv2_joy/common/irs/getservent_r.c
+++ b/usr/src/lib/libresolv2_joy/common/irs/getservent_r.c
@@ -15,10 +15,6 @@
  * OF OR IN CONNECTION WITH THE USE OR PERFORMANCE OF THIS SOFTWARE.
  */
 
-#if defined(LIBC_SCCS) && !defined(lint)
-static const char rcsid[] = "$Id: getservent_r.c,v 1.6 2006/08/01 01:14:16 marka Exp $";
-#endif /* LIBC_SCCS and not lint */
-
 #include <port_before.h>
 #if !defined(_REENTRANT) || !defined(DO_PTHREADS)
 	static int getservent_r_not_required = 0;
diff --git a/usr/src/lib/libresolv2_joy/common/irs/hesiod.c b/usr/src/lib/libresolv2_joy/common/irs/hesiod.c
index 7641bf80ac..2dd05431a5 100644
--- a/usr/src/lib/libresolv2_joy/common/irs/hesiod.c
+++ b/usr/src/lib/libresolv2_joy/common/irs/hesiod.c
@@ -1,7 +1,3 @@
-#if defined(LIBC_SCCS) && !defined(lint)
-static const char rcsid[] = "$Id: hesiod.c,v 1.7 2005/07/28 06:51:48 marka Exp $";
-#endif
-
 /*
  * Copyright (c) 2004 by Internet Systems Consortium, Inc. ("ISC")
  * Copyright (c) 1996,1999 by Internet Software Consortium.
diff --git a/usr/src/lib/libresolv2_joy/common/irs/irp.c b/usr/src/lib/libresolv2_joy/common/irs/irp.c
index ef10631c22..07659c49af 100644
--- a/usr/src/lib/libresolv2_joy/common/irs/irp.c
+++ b/usr/src/lib/libresolv2_joy/common/irs/irp.c
@@ -15,12 +15,6 @@
  * PERFORMANCE OF THIS SOFTWARE.
  */
 
-#if !defined(LINT) && !defined(CODECENTER)
-static const char rcsid[] = "$Id: irp.c,v 1.12 2008/11/14 02:36:51 marka Exp $";
-#endif
-
-/* Imports */
-
 #include "port_before.h"
 
 #include <syslog.h>
diff --git a/usr/src/lib/libresolv2_joy/common/irs/irp_ho.c b/usr/src/lib/libresolv2_joy/common/irs/irp_ho.c
index b0de31dc89..7a13d2dfc6 100644
--- a/usr/src/lib/libresolv2_joy/common/irs/irp_ho.c
+++ b/usr/src/lib/libresolv2_joy/common/irs/irp_ho.c
@@ -15,10 +15,6 @@
  * OF OR IN CONNECTION WITH THE USE OR PERFORMANCE OF THIS SOFTWARE.
  */
 
-#if defined(LIBC_SCCS) && !defined(lint)
-static const char rcsid[] = "$Id: irp_ho.c,v 1.3 2005/04/27 04:56:28 sra Exp $";
-#endif /* LIBC_SCCS and not lint */
-
 /* Imports. */
 
 #include "port_before.h"
diff --git a/usr/src/lib/libresolv2_joy/common/irs/irp_ng.c b/usr/src/lib/libresolv2_joy/common/irs/irp_ng.c
index 1af862cab4..ef5e5db67c 100644
--- a/usr/src/lib/libresolv2_joy/common/irs/irp_ng.c
+++ b/usr/src/lib/libresolv2_joy/common/irs/irp_ng.c
@@ -15,10 +15,6 @@
  * OF OR IN CONNECTION WITH THE USE OR PERFORMANCE OF THIS SOFTWARE.
  */
 
-#if !defined(LINT) && !defined(CODECENTER)
-static const char rcsid[] = "$Id: irp_ng.c,v 1.4 2006/12/07 04:46:27 marka Exp $";
-#endif
-
 /* Imports */
 
 #include "port_before.h"
diff --git a/usr/src/lib/libresolv2_joy/common/irs/irp_nw.c b/usr/src/lib/libresolv2_joy/common/irs/irp_nw.c
index 3f2381f95d..d032f6a608 100644
--- a/usr/src/lib/libresolv2_joy/common/irs/irp_nw.c
+++ b/usr/src/lib/libresolv2_joy/common/irs/irp_nw.c
@@ -15,14 +15,6 @@
  * OF OR IN CONNECTION WITH THE USE OR PERFORMANCE OF THIS SOFTWARE.
  */
 
-#if defined(LIBC_SCCS) && !defined(lint)
-static const char rcsid[] = "$Id: irp_nw.c,v 1.4 2006/03/09 23:57:56 marka Exp $";
-#endif /* LIBC_SCCS and not lint */
-
-#if 0
-
-#endif
-
 /* Imports */
 
 #include "port_before.h"
diff --git a/usr/src/lib/libresolv2_joy/common/irs/irp_pr.c b/usr/src/lib/libresolv2_joy/common/irs/irp_pr.c
index ea876e8281..a78f5b2b73 100644
--- a/usr/src/lib/libresolv2_joy/common/irs/irp_pr.c
+++ b/usr/src/lib/libresolv2_joy/common/irs/irp_pr.c
@@ -15,12 +15,6 @@
  * OF OR IN CONNECTION WITH THE USE OR PERFORMANCE OF THIS SOFTWARE.
  */
 
-#if defined(LIBC_SCCS) && !defined(lint)
-static const char rcsid[] = "$Id: irp_pr.c,v 1.3 2005/04/27 04:56:29 sra Exp $";
-#endif /* LIBC_SCCS and not lint */
-
-/* extern */
-
 #include "port_before.h"
 
 #include <syslog.h>
diff --git a/usr/src/lib/libresolv2_joy/common/irs/irp_sv.c b/usr/src/lib/libresolv2_joy/common/irs/irp_sv.c
index 577e697fe6..5cd3e74032 100644
--- a/usr/src/lib/libresolv2_joy/common/irs/irp_sv.c
+++ b/usr/src/lib/libresolv2_joy/common/irs/irp_sv.c
@@ -15,12 +15,6 @@
  * OF OR IN CONNECTION WITH THE USE OR PERFORMANCE OF THIS SOFTWARE.
  */
 
-#if defined(LIBC_SCCS) && !defined(lint)
-static const char rcsid[] = "$Id: irp_sv.c,v 1.3 2005/04/27 04:56:29 sra Exp $";
-#endif /* LIBC_SCCS and not lint */
-
-/* extern */
-
 #include "port_before.h"
 
 #include <syslog.h>
diff --git a/usr/src/lib/libresolv2_joy/common/irs/irpmarshall.c b/usr/src/lib/libresolv2_joy/common/irs/irpmarshall.c
index 85ffff1866..a33bb5bfbd 100644
--- a/usr/src/lib/libresolv2_joy/common/irs/irpmarshall.c
+++ b/usr/src/lib/libresolv2_joy/common/irs/irpmarshall.c
@@ -48,10 +48,6 @@
  * OF OR IN CONNECTION WITH THE USE OR PERFORMANCE OF THIS SOFTWARE.
  */
 
-#if defined(LIBC_SCCS) && !defined(lint)
-static const char rcsid[] = "$Id: irpmarshall.c,v 1.7 2006/03/09 23:57:56 marka Exp $";
-#endif /* LIBC_SCCS and not lint */
-
 #if 0
 
 Check values are in approrpriate endian order.
diff --git a/usr/src/lib/libresolv2_joy/common/irs/irs_data.c b/usr/src/lib/libresolv2_joy/common/irs/irs_data.c
index f12ca20f38..3087b1189d 100644
--- a/usr/src/lib/libresolv2_joy/common/irs/irs_data.c
+++ b/usr/src/lib/libresolv2_joy/common/irs/irs_data.c
@@ -19,10 +19,6 @@
  * OF OR IN CONNECTION WITH THE USE OR PERFORMANCE OF THIS SOFTWARE.
  */
 
-#if !defined(LINT) && !defined(CODECENTER)
-static const char rcsid[] = "$Id: irs_data.c,v 1.12 2007/08/27 03:32:26 marka Exp $";
-#endif
-
 #include "port_before.h"
 
 #ifndef __BIND_NOSTATIC
diff --git a/usr/src/lib/libresolv2_joy/common/irs/lcl.c b/usr/src/lib/libresolv2_joy/common/irs/lcl.c
index 9dd6967b87..3a34b75c6d 100644
--- a/usr/src/lib/libresolv2_joy/common/irs/lcl.c
+++ b/usr/src/lib/libresolv2_joy/common/irs/lcl.c
@@ -15,12 +15,6 @@
  * OF OR IN CONNECTION WITH THE USE OR PERFORMANCE OF THIS SOFTWARE.
  */
 
-#if !defined(LINT) && !defined(CODECENTER)
-static const char rcsid[] = "$Id: lcl.c,v 1.4 2005/04/27 04:56:30 sra Exp $";
-#endif
-
-/* Imports */
-
 #include "port_before.h"
 
 #include <stdlib.h>
diff --git a/usr/src/lib/libresolv2_joy/common/irs/lcl_ho.c b/usr/src/lib/libresolv2_joy/common/irs/lcl_ho.c
index 17c9a5e725..e8893bf095 100644
--- a/usr/src/lib/libresolv2_joy/common/irs/lcl_ho.c
+++ b/usr/src/lib/libresolv2_joy/common/irs/lcl_ho.c
@@ -51,10 +51,6 @@
 /* from gethostnamadr.c	8.1 (Berkeley) 6/4/93 */
 /* BIND Id: gethnamaddr.c,v 8.15 1996/05/22 04:56:30 vixie Exp $ */
 
-#if defined(LIBC_SCCS) && !defined(lint)
-static const char rcsid[] = "$Id: lcl_ho.c,v 1.5 2006/03/09 23:57:56 marka Exp $";
-#endif /* LIBC_SCCS and not lint */
-
 /* Imports. */
 
 #include "port_before.h"
diff --git a/usr/src/lib/libresolv2_joy/common/irs/lcl_ng.c b/usr/src/lib/libresolv2_joy/common/irs/lcl_ng.c
index 319725ce70..089a3c437e 100644
--- a/usr/src/lib/libresolv2_joy/common/irs/lcl_ng.c
+++ b/usr/src/lib/libresolv2_joy/common/irs/lcl_ng.c
@@ -15,10 +15,6 @@
  * OF OR IN CONNECTION WITH THE USE OR PERFORMANCE OF THIS SOFTWARE.
  */
 
-#if !defined(LINT) && !defined(CODECENTER)
-static const char rcsid[] = "$Id: lcl_ng.c,v 1.3 2005/04/27 04:56:31 sra Exp $";
-#endif
-
 /* Imports */
 
 #include "port_before.h"
diff --git a/usr/src/lib/libresolv2_joy/common/irs/lcl_nw.c b/usr/src/lib/libresolv2_joy/common/irs/lcl_nw.c
index 2e5cd55a6c..e1c5837944 100644
--- a/usr/src/lib/libresolv2_joy/common/irs/lcl_nw.c
+++ b/usr/src/lib/libresolv2_joy/common/irs/lcl_nw.c
@@ -48,11 +48,8 @@
  * OF OR IN CONNECTION WITH THE USE OR PERFORMANCE OF THIS SOFTWARE.
  */
 
-#if defined(LIBC_SCCS) && !defined(lint)
-static const char rcsid[] = "$Id: lcl_nw.c,v 1.4 2005/04/27 04:56:31 sra Exp $";
 /* from getgrent.c 8.2 (Berkeley) 3/21/94"; */
 /* from BSDI Id: getgrent.c,v 2.8 1996/05/28 18:15:14 bostic Exp $	*/
-#endif /* LIBC_SCCS and not lint */
 
 /* Imports */
 
diff --git a/usr/src/lib/libresolv2_joy/common/irs/lcl_pr.c b/usr/src/lib/libresolv2_joy/common/irs/lcl_pr.c
index e1538530eb..22720f3e99 100644
--- a/usr/src/lib/libresolv2_joy/common/irs/lcl_pr.c
+++ b/usr/src/lib/libresolv2_joy/common/irs/lcl_pr.c
@@ -48,12 +48,6 @@
  * OF OR IN CONNECTION WITH THE USE OR PERFORMANCE OF THIS SOFTWARE.
  */
 
-#if defined(LIBC_SCCS) && !defined(lint)
-static const char rcsid[] = "$Id: lcl_pr.c,v 1.4 2006/03/09 23:57:56 marka Exp $";
-#endif /* LIBC_SCCS and not lint */
-
-/* extern */
-
 #include "port_before.h"
 
 #include <sys/types.h>
diff --git a/usr/src/lib/libresolv2_joy/common/irs/lcl_sv.c b/usr/src/lib/libresolv2_joy/common/irs/lcl_sv.c
index ad6526430c..a1ef71be2a 100644
--- a/usr/src/lib/libresolv2_joy/common/irs/lcl_sv.c
+++ b/usr/src/lib/libresolv2_joy/common/irs/lcl_sv.c
@@ -48,12 +48,6 @@
  * OF OR IN CONNECTION WITH THE USE OR PERFORMANCE OF THIS SOFTWARE.
  */
 
-#if defined(LIBC_SCCS) && !defined(lint)
-static const char rcsid[] = "$Id: lcl_sv.c,v 1.4 2005/04/27 04:56:31 sra Exp $";
-#endif /* LIBC_SCCS and not lint */
-
-/* extern */
-
 #include "port_before.h"
 
 #include <sys/types.h>
diff --git a/usr/src/lib/libresolv2_joy/common/irs/nis.c b/usr/src/lib/libresolv2_joy/common/irs/nis.c
index ac1796543c..0325c47a1c 100644
--- a/usr/src/lib/libresolv2_joy/common/irs/nis.c
+++ b/usr/src/lib/libresolv2_joy/common/irs/nis.c
@@ -15,12 +15,6 @@
  * OF OR IN CONNECTION WITH THE USE OR PERFORMANCE OF THIS SOFTWARE.
  */
 
-#if defined(LIBC_SCCS) && !defined(lint)
-static const char rcsid[] = "$Id: nis.c,v 1.3 2005/04/27 04:56:32 sra Exp $";
-#endif
-
-/* Imports */
-
 #include "port_before.h"
 
 #ifdef WANT_IRS_NIS
diff --git a/usr/src/lib/libresolv2_joy/common/irs/nul_ng.c b/usr/src/lib/libresolv2_joy/common/irs/nul_ng.c
index 504813bde1..a16ff9c2f0 100644
--- a/usr/src/lib/libresolv2_joy/common/irs/nul_ng.c
+++ b/usr/src/lib/libresolv2_joy/common/irs/nul_ng.c
@@ -15,10 +15,6 @@
  * OF OR IN CONNECTION WITH THE USE OR PERFORMANCE OF THIS SOFTWARE.
  */
 
-#if defined(LIBC_SCCS) && !defined(lint)
-static const char rcsid[] = "$Id: nul_ng.c,v 1.3 2005/04/27 04:56:34 sra Exp $";
-#endif
-
 /*! \file
  * \brief
  * nul_ng.c - the netgroup accessor null map
diff --git a/usr/src/lib/libresolv2_joy/common/irs/util.c b/usr/src/lib/libresolv2_joy/common/irs/util.c
index 9c70eabddc..4afc530fb0 100644
--- a/usr/src/lib/libresolv2_joy/common/irs/util.c
+++ b/usr/src/lib/libresolv2_joy/common/irs/util.c
@@ -15,10 +15,6 @@
  * OF OR IN CONNECTION WITH THE USE OR PERFORMANCE OF THIS SOFTWARE.
  */
 
-#if defined(LIBC_SCCS) && !defined(lint)
-static const char rcsid[] = "$Id: util.c,v 1.3 2005/04/27 04:56:34 sra Exp $";
-#endif
-
 #include "port_before.h"
 
 #include <sys/types.h>
diff --git a/usr/src/lib/libresolv2_joy/common/isc/assertions.c b/usr/src/lib/libresolv2_joy/common/isc/assertions.c
index b71e5a32d3..6516d66cc8 100644
--- a/usr/src/lib/libresolv2_joy/common/isc/assertions.c
+++ b/usr/src/lib/libresolv2_joy/common/isc/assertions.c
@@ -15,10 +15,6 @@
  * PERFORMANCE OF THIS SOFTWARE.
  */
 
-#if !defined(LINT) && !defined(CODECENTER)
-static const char rcsid[] = "$Id: assertions.c,v 1.5 2008/11/14 02:36:51 marka Exp $";
-#endif
-
 #include "port_before.h"
 
 #include <errno.h>
diff --git a/usr/src/lib/libresolv2_joy/common/isc/base64.c b/usr/src/lib/libresolv2_joy/common/isc/base64.c
index 79c35722b1..a819fd97bf 100644
--- a/usr/src/lib/libresolv2_joy/common/isc/base64.c
+++ b/usr/src/lib/libresolv2_joy/common/isc/base64.c
@@ -46,10 +46,6 @@
  * IF IBM IS APPRISED OF THE POSSIBILITY OF SUCH DAMAGES.
  */
 
-#if !defined(LINT) && !defined(CODECENTER)
-static const char rcsid[] = "$Id: base64.c,v 1.4 2005/04/27 04:56:34 sra Exp $";
-#endif /* not lint */
-
 #include "port_before.h"
 
 #include <sys/types.h>
diff --git a/usr/src/lib/libresolv2_joy/common/isc/bitncmp.c b/usr/src/lib/libresolv2_joy/common/isc/bitncmp.c
index efe5009292..d229e04c89 100644
--- a/usr/src/lib/libresolv2_joy/common/isc/bitncmp.c
+++ b/usr/src/lib/libresolv2_joy/common/isc/bitncmp.c
@@ -15,10 +15,6 @@
  * PERFORMANCE OF THIS SOFTWARE.
  */
 
-#if defined(LIBC_SCCS) && !defined(lint)
-static const char rcsid[] = "$Id: bitncmp.c,v 1.5 2008/11/14 02:36:51 marka Exp $";
-#endif
-
 #include "port_before.h"
 
 #include <sys/types.h>
diff --git a/usr/src/lib/libresolv2_joy/common/isc/ctl_clnt.c b/usr/src/lib/libresolv2_joy/common/isc/ctl_clnt.c
index f71001a6d4..eca06b2ee2 100644
--- a/usr/src/lib/libresolv2_joy/common/isc/ctl_clnt.c
+++ b/usr/src/lib/libresolv2_joy/common/isc/ctl_clnt.c
@@ -15,12 +15,6 @@
  * PERFORMANCE OF THIS SOFTWARE.
  */
 
-#if !defined(lint) && !defined(SABER)
-static const char rcsid[] = "$Id: ctl_clnt.c,v 1.11 2008/11/14 02:36:51 marka Exp $";
-#endif /* not lint */
-
-/* Extern. */
-
 #include "port_before.h"
 
 #include <sys/param.h>
diff --git a/usr/src/lib/libresolv2_joy/common/isc/ctl_p.c b/usr/src/lib/libresolv2_joy/common/isc/ctl_p.c
index 7ab719a5e6..3c0ebf4037 100644
--- a/usr/src/lib/libresolv2_joy/common/isc/ctl_p.c
+++ b/usr/src/lib/libresolv2_joy/common/isc/ctl_p.c
@@ -1,7 +1,3 @@
-#if !defined(lint) && !defined(SABER)
-static const char rcsid[] = "$Id: ctl_p.c,v 1.4 2005/04/27 04:56:35 sra Exp $";
-#endif /* not lint */
-
 /*
  * Copyright (c) 2004 by Internet Systems Consortium, Inc. ("ISC")
  * Copyright (c) 1998,1999 by Internet Software Consortium.
diff --git a/usr/src/lib/libresolv2_joy/common/isc/ctl_srvr.c b/usr/src/lib/libresolv2_joy/common/isc/ctl_srvr.c
index 8fd7a21ffa..12191cf0b0 100644
--- a/usr/src/lib/libresolv2_joy/common/isc/ctl_srvr.c
+++ b/usr/src/lib/libresolv2_joy/common/isc/ctl_srvr.c
@@ -15,12 +15,6 @@
  * PERFORMANCE OF THIS SOFTWARE.
  */
 
-#if !defined(lint) && !defined(SABER)
-static const char rcsid[] = "$Id: ctl_srvr.c,v 1.10 2008/11/14 02:36:51 marka Exp $";
-#endif /* not lint */
-
-/* Extern. */
-
 #include "port_before.h"
 
 #include <sys/param.h>
diff --git a/usr/src/lib/libresolv2_joy/common/isc/ev_connects.c b/usr/src/lib/libresolv2_joy/common/isc/ev_connects.c
index 38dfdbe512..821eb8ebff 100644
--- a/usr/src/lib/libresolv2_joy/common/isc/ev_connects.c
+++ b/usr/src/lib/libresolv2_joy/common/isc/ev_connects.c
@@ -19,12 +19,6 @@
  * vix 16sep96 [initial]
  */
 
-#if !defined(LINT) && !defined(CODECENTER)
-static const char rcsid[] = "$Id: ev_connects.c,v 1.8 2006/03/09 23:57:56 marka Exp $";
-#endif
-
-/* Import. */
-
 #include "port_before.h"
 #include "fd_setsize.h"
 
diff --git a/usr/src/lib/libresolv2_joy/common/isc/ev_files.c b/usr/src/lib/libresolv2_joy/common/isc/ev_files.c
index b12baf1aaa..f75ff6b646 100644
--- a/usr/src/lib/libresolv2_joy/common/isc/ev_files.c
+++ b/usr/src/lib/libresolv2_joy/common/isc/ev_files.c
@@ -19,10 +19,6 @@
  * vix 11sep95 [initial]
  */
 
-#if !defined(LINT) && !defined(CODECENTER)
-static const char rcsid[] = "$Id: ev_files.c,v 1.8 2005/07/28 06:51:48 marka Exp $";
-#endif
-
 #include "port_before.h"
 #include "fd_setsize.h"
 
diff --git a/usr/src/lib/libresolv2_joy/common/isc/ev_streams.c b/usr/src/lib/libresolv2_joy/common/isc/ev_streams.c
index 5dad36d04a..824b4e25c1 100644
--- a/usr/src/lib/libresolv2_joy/common/isc/ev_streams.c
+++ b/usr/src/lib/libresolv2_joy/common/isc/ev_streams.c
@@ -19,10 +19,6 @@
  * vix 04mar96 [initial]
  */
 
-#if !defined(LINT) && !defined(CODECENTER)
-static const char rcsid[] = "$Id: ev_streams.c,v 1.5 2005/04/27 04:56:36 sra Exp $";
-#endif
-
 #include "port_before.h"
 #include "fd_setsize.h"
 
diff --git a/usr/src/lib/libresolv2_joy/common/isc/ev_timers.c b/usr/src/lib/libresolv2_joy/common/isc/ev_timers.c
index 12ac2cebca..4aab57f5ff 100644
--- a/usr/src/lib/libresolv2_joy/common/isc/ev_timers.c
+++ b/usr/src/lib/libresolv2_joy/common/isc/ev_timers.c
@@ -19,12 +19,6 @@
  * vix 09sep95 [initial]
  */
 
-#if !defined(LINT) && !defined(CODECENTER)
-static const char rcsid[] = "$Id: ev_timers.c,v 1.6 2005/04/27 04:56:36 sra Exp $";
-#endif
-
-/* Import. */
-
 #include "port_before.h"
 #include "fd_setsize.h"
 
diff --git a/usr/src/lib/libresolv2_joy/common/isc/ev_waits.c b/usr/src/lib/libresolv2_joy/common/isc/ev_waits.c
index 99da1526c7..9c4e819eb8 100644
--- a/usr/src/lib/libresolv2_joy/common/isc/ev_waits.c
+++ b/usr/src/lib/libresolv2_joy/common/isc/ev_waits.c
@@ -19,10 +19,6 @@
  * vix 05dec95 [initial]
  */
 
-#if !defined(LINT) && !defined(CODECENTER)
-static const char rcsid[] = "$Id: ev_waits.c,v 1.4 2005/04/27 04:56:36 sra Exp $";
-#endif
-
 #include "port_before.h"
 #include "fd_setsize.h"
 
diff --git a/usr/src/lib/libresolv2_joy/common/isc/eventlib.c b/usr/src/lib/libresolv2_joy/common/isc/eventlib.c
index be4a7848b9..f25c2098ab 100644
--- a/usr/src/lib/libresolv2_joy/common/isc/eventlib.c
+++ b/usr/src/lib/libresolv2_joy/common/isc/eventlib.c
@@ -1,6 +1,7 @@
 /*
  * Copyright (c) 2004 by Internet Systems Consortium, Inc. ("ISC")
  * Copyright (c) 1995-1999 by Internet Software Consortium
+ * Copyright 2018 Joyent, Inc.
  *
  * Permission to use, copy, modify, and distribute this software for any
  * purpose with or without fee is hereby granted, provided that the above
@@ -19,10 +20,6 @@
  * vix 09sep95 [initial]
  */
 
-#if !defined(LINT) && !defined(CODECENTER)
-static const char rcsid[] = "$Id: eventlib.c,v 1.10 2006/03/09 23:57:56 marka Exp $";
-#endif
-
 #include "port_before.h"
 #include "fd_setsize.h"
 
@@ -737,7 +734,7 @@ pselect(int nfds, void *rfds, void *wfds, void *efds,
 	struct timespec *tsp,
 	const sigset_t *sigmask)
 {
-	struct timeval tv, *tvp;
+	struct timeval tv;
 	sigset_t sigs;
 	int n;
 #ifdef USE_POLL
@@ -747,16 +744,18 @@ pselect(int nfds, void *rfds, void *wfds, void *efds,
 	nfds_t		pnfds;
 
 	UNUSED(nfds);
+#else
+	struct timeval *tvp = NULL;
 #endif /* USE_POLL */
 
 	if (tsp) {
-		tvp = &tv;
 		tv = evTimeVal(*tsp);
 #ifdef USE_POLL
 		polltimeout = 1000 * tv.tv_sec + tv.tv_usec / 1000;
+#else
+		tvp = &tv;
 #endif /* USE_POLL */
-	} else
-		tvp = NULL;
+	}
 	if (sigmask)
 		sigprocmask(SIG_SETMASK, sigmask, &sigs);
 #ifndef USE_POLL
diff --git a/usr/src/lib/libresolv2_joy/common/isc/heap.c b/usr/src/lib/libresolv2_joy/common/isc/heap.c
index 3d22b6fc71..3e48d244e2 100644
--- a/usr/src/lib/libresolv2_joy/common/isc/heap.c
+++ b/usr/src/lib/libresolv2_joy/common/isc/heap.c
@@ -25,10 +25,6 @@
  *	ISBN 0-201-06673-4, chapter 11.
  */
 
-#if !defined(LINT) && !defined(CODECENTER)
-static const char rcsid[] = "$Id: heap.c,v 1.4 2006/03/09 23:57:56 marka Exp $";
-#endif /* not lint */
-
 #include "port_before.h"
 
 #include <stddef.h>
diff --git a/usr/src/lib/libresolv2_joy/common/isc/logging.c b/usr/src/lib/libresolv2_joy/common/isc/logging.c
index 8c2af2b9e3..ac4806b4f8 100644
--- a/usr/src/lib/libresolv2_joy/common/isc/logging.c
+++ b/usr/src/lib/libresolv2_joy/common/isc/logging.c
@@ -15,10 +15,6 @@
  * PERFORMANCE OF THIS SOFTWARE.
  */
 
-#if !defined(LINT) && !defined(CODECENTER)
-static const char rcsid[] = "$Id: logging.c,v 1.9 2008/11/14 02:36:51 marka Exp $";
-#endif /* not lint */
-
 #include "port_before.h"
 
 #include <sys/types.h>
diff --git a/usr/src/lib/libresolv2_joy/common/isc/memcluster.c b/usr/src/lib/libresolv2_joy/common/isc/memcluster.c
index 515793fd6a..1a2db0605b 100644
--- a/usr/src/lib/libresolv2_joy/common/isc/memcluster.c
+++ b/usr/src/lib/libresolv2_joy/common/isc/memcluster.c
@@ -22,11 +22,6 @@
 /* #define DEBUGGING_MEMCLUSTER */
 #define MEMCLUSTER_ATEND
 
-
-#if !defined(LINT) && !defined(CODECENTER)
-static const char rcsid[] = "$Id: memcluster.c,v 1.11 2006/08/30 23:34:38 marka Exp $";
-#endif /* not lint */
-
 #include "port_before.h"
 
 #include <sys/types.h>
diff --git a/usr/src/lib/libresolv2_joy/common/isc/tree.c b/usr/src/lib/libresolv2_joy/common/isc/tree.c
index 8ba675fbe8..404014602a 100644
--- a/usr/src/lib/libresolv2_joy/common/isc/tree.c
+++ b/usr/src/lib/libresolv2_joy/common/isc/tree.c
@@ -1,7 +1,3 @@
-#ifndef LINT
-static const char rcsid[] = "$Id: tree.c,v 1.4 2005/04/27 04:56:39 sra Exp $";
-#endif
-
 /*%
  * tree - balanced binary tree library
  *
diff --git a/usr/src/lib/libresolv2_joy/common/nameser/ns_date.c b/usr/src/lib/libresolv2_joy/common/nameser/ns_date.c
index 292375af63..5802172a03 100644
--- a/usr/src/lib/libresolv2_joy/common/nameser/ns_date.c
+++ b/usr/src/lib/libresolv2_joy/common/nameser/ns_date.c
@@ -15,12 +15,6 @@
  * OF OR IN CONNECTION WITH THE USE OR PERFORMANCE OF THIS SOFTWARE.
  */
 
-#ifndef lint
-static const char rcsid[] = "$Id: ns_date.c,v 1.6 2005/04/27 04:56:39 sra Exp $";
-#endif
-
-/* Import. */
-
 #include "port_before.h"
 
 #include <arpa/nameser.h>
diff --git a/usr/src/lib/libresolv2_joy/common/nameser/ns_name.c b/usr/src/lib/libresolv2_joy/common/nameser/ns_name.c
index f6b0accef1..a2bf99f1c1 100644
--- a/usr/src/lib/libresolv2_joy/common/nameser/ns_name.c
+++ b/usr/src/lib/libresolv2_joy/common/nameser/ns_name.c
@@ -15,10 +15,6 @@
  * OF OR IN CONNECTION WITH THE USE OR PERFORMANCE OF THIS SOFTWARE.
  */
 
-#ifndef lint
-static const char rcsid[] = "$Id: ns_name.c,v 1.11 2009/01/23 19:59:16 each Exp $";
-#endif
-
 #include "port_before.h"
 
 #include <sys/types.h>
diff --git a/usr/src/lib/libresolv2_joy/common/nameser/ns_netint.c b/usr/src/lib/libresolv2_joy/common/nameser/ns_netint.c
index e196217f9b..f57edeb8fd 100644
--- a/usr/src/lib/libresolv2_joy/common/nameser/ns_netint.c
+++ b/usr/src/lib/libresolv2_joy/common/nameser/ns_netint.c
@@ -15,12 +15,6 @@
  * OF OR IN CONNECTION WITH THE USE OR PERFORMANCE OF THIS SOFTWARE.
  */
 
-#ifndef lint
-static const char rcsid[] = "$Id: ns_netint.c,v 1.3 2005/04/27 04:56:40 sra Exp $";
-#endif
-
-/* Import. */
-
 #include "port_before.h"
 
 #include <arpa/nameser.h>
diff --git a/usr/src/lib/libresolv2_joy/common/nameser/ns_newmsg.c b/usr/src/lib/libresolv2_joy/common/nameser/ns_newmsg.c
index c18dd060e1..870a4608e2 100644
--- a/usr/src/lib/libresolv2_joy/common/nameser/ns_newmsg.c
+++ b/usr/src/lib/libresolv2_joy/common/nameser/ns_newmsg.c
@@ -20,10 +20,6 @@
  * PERFORMANCE OF THIS SOFTWARE.
  */
 
-#ifndef lint
-static const char rcsid[] = "$Id: ns_newmsg.c,v 1.3 2009/02/26 10:48:57 marka Exp $";
-#endif
-
 #include <port_before.h>
 
 #include <arpa/nameser.h>
diff --git a/usr/src/lib/libresolv2_joy/common/nameser/ns_parse.c b/usr/src/lib/libresolv2_joy/common/nameser/ns_parse.c
index ba11eea707..a8909c2984 100644
--- a/usr/src/lib/libresolv2_joy/common/nameser/ns_parse.c
+++ b/usr/src/lib/libresolv2_joy/common/nameser/ns_parse.c
@@ -15,12 +15,6 @@
  * OF OR IN CONNECTION WITH THE USE OR PERFORMANCE OF THIS SOFTWARE.
  */
 
-#ifndef lint
-static const char rcsid[] = "$Id: ns_parse.c,v 1.10 2009/01/23 19:59:16 each Exp $";
-#endif
-
-/* Import. */
-
 #include "port_before.h"
 
 #include <sys/types.h>
diff --git a/usr/src/lib/libresolv2_joy/common/nameser/ns_print.c b/usr/src/lib/libresolv2_joy/common/nameser/ns_print.c
index e70df376c1..2624c63732 100644
--- a/usr/src/lib/libresolv2_joy/common/nameser/ns_print.c
+++ b/usr/src/lib/libresolv2_joy/common/nameser/ns_print.c
@@ -1,6 +1,7 @@
 /*
  * Copyright (c) 2004 by Internet Systems Consortium, Inc. ("ISC")
  * Copyright (c) 1996-1999 by Internet Software Consortium.
+ * Copyright 2018 Joyent, Inc.
  *
  * Permission to use, copy, modify, and distribute this software for any
  * purpose with or without fee is hereby granted, provided that the above
@@ -15,12 +16,6 @@
  * OF OR IN CONNECTION WITH THE USE OR PERFORMANCE OF THIS SOFTWARE.
  */
 
-#ifndef lint
-static const char rcsid[] = "$Id: ns_print.c,v 1.12 2009/03/03 05:29:58 each Exp $";
-#endif
-
-/* Import. */
-
 #include "port_before.h"
 
 #include <sys/types.h>
@@ -900,6 +895,7 @@ ns_sprintrrf(const u_char *msg, size_t msglen,
 			if (len > 15)
 				T(addstr(" )", 2, &buf, &buflen));
 		}
+		break;
 	}
 
 	case ns_t_ipseckey: {
@@ -991,6 +987,7 @@ ns_sprintrrf(const u_char *msg, size_t msglen,
 			if (len > 15)
 				T(addstr(" )", 2, &buf, &buflen));
 		}
+		break;
 	}
 
 	case ns_t_hip: {
diff --git a/usr/src/lib/libresolv2_joy/common/nameser/ns_rdata.c b/usr/src/lib/libresolv2_joy/common/nameser/ns_rdata.c
index eac4052278..76617de9ad 100644
--- a/usr/src/lib/libresolv2_joy/common/nameser/ns_rdata.c
+++ b/usr/src/lib/libresolv2_joy/common/nameser/ns_rdata.c
@@ -20,10 +20,6 @@
  * PERFORMANCE OF THIS SOFTWARE.
  */
 
-#ifndef lint
-static const char rcsid[] = "$Id: ns_rdata.c,v 1.2 2009/01/23 23:49:15 tbox Exp $";
-#endif
-
 #include "port_before.h"
 
 #if __OpenBSD__
diff --git a/usr/src/lib/libresolv2_joy/common/nameser/ns_samedomain.c b/usr/src/lib/libresolv2_joy/common/nameser/ns_samedomain.c
index 5e9f5cab54..b4c4ae68db 100644
--- a/usr/src/lib/libresolv2_joy/common/nameser/ns_samedomain.c
+++ b/usr/src/lib/libresolv2_joy/common/nameser/ns_samedomain.c
@@ -15,10 +15,6 @@
  * OF OR IN CONNECTION WITH THE USE OR PERFORMANCE OF THIS SOFTWARE.
  */
 
-#ifndef lint
-static const char rcsid[] = "$Id: ns_samedomain.c,v 1.6 2005/04/27 04:56:40 sra Exp $";
-#endif
-
 #include "port_before.h"
 
 #include <sys/types.h>
diff --git a/usr/src/lib/libresolv2_joy/common/nameser/ns_sign.c b/usr/src/lib/libresolv2_joy/common/nameser/ns_sign.c
index fc7dd19f7b..4db8d33679 100644
--- a/usr/src/lib/libresolv2_joy/common/nameser/ns_sign.c
+++ b/usr/src/lib/libresolv2_joy/common/nameser/ns_sign.c
@@ -15,12 +15,6 @@
  * OF OR IN CONNECTION WITH THE USE OR PERFORMANCE OF THIS SOFTWARE.
  */
 
-#ifndef lint
-static const char rcsid[] = "$Id: ns_sign.c,v 1.6 2006/03/09 23:57:56 marka Exp $";
-#endif
-
-/* Import. */
-
 #include "port_before.h"
 #include "fd_setsize.h"
 
diff --git a/usr/src/lib/libresolv2_joy/common/nameser/ns_ttl.c b/usr/src/lib/libresolv2_joy/common/nameser/ns_ttl.c
index 69c2f83f57..635a11905e 100644
--- a/usr/src/lib/libresolv2_joy/common/nameser/ns_ttl.c
+++ b/usr/src/lib/libresolv2_joy/common/nameser/ns_ttl.c
@@ -1,6 +1,7 @@
 /*
  * Copyright (c) 2004 by Internet Systems Consortium, Inc. ("ISC")
  * Copyright (c) 1996,1999 by Internet Software Consortium.
+ * Copyright 2018 Joyent, Inc.
  *
  * Permission to use, copy, modify, and distribute this software for any
  * purpose with or without fee is hereby granted, provided that the above
@@ -15,12 +16,6 @@
  * OF OR IN CONNECTION WITH THE USE OR PERFORMANCE OF THIS SOFTWARE.
  */
 
-#ifndef lint
-static const char rcsid[] = "$Id: ns_ttl.c,v 1.4 2005/07/28 06:51:49 marka Exp $";
-#endif
-
-/* Import. */
-
 #include "port_before.h"
 
 #include <arpa/nameser.h>
@@ -117,9 +112,13 @@ ns_parse_ttl(const char *src, u_long *dst) {
 			ch = toupper(ch);
 		switch (ch) {
 		case 'W':  tmp *= 7;
+			/* FALLTHROUGH */
 		case 'D':  tmp *= 24;
+			/* FALLTHROUGH */
 		case 'H':  tmp *= 60;
+			/* FALLTHROUGH */
 		case 'M':  tmp *= 60;
+			/* FALLTHROUGH */
 		case 'S':  break;
 		default:   goto einval;
 		}
diff --git a/usr/src/lib/libresolv2_joy/common/nameser/ns_verify.c b/usr/src/lib/libresolv2_joy/common/nameser/ns_verify.c
index b4b63d4641..93d8d59865 100644
--- a/usr/src/lib/libresolv2_joy/common/nameser/ns_verify.c
+++ b/usr/src/lib/libresolv2_joy/common/nameser/ns_verify.c
@@ -15,12 +15,6 @@
  * OF OR IN CONNECTION WITH THE USE OR PERFORMANCE OF THIS SOFTWARE.
  */
 
-#ifndef lint
-static const char rcsid[] = "$Id: ns_verify.c,v 1.5 2006/03/09 23:57:56 marka Exp $";
-#endif
-
-/* Import. */
-
 #include "port_before.h"
 #include "fd_setsize.h"
 
diff --git a/usr/src/lib/libresolv2_joy/common/resolv/herror.c b/usr/src/lib/libresolv2_joy/common/resolv/herror.c
index 568ce3ce68..5181c5792c 100644
--- a/usr/src/lib/libresolv2_joy/common/resolv/herror.c
+++ b/usr/src/lib/libresolv2_joy/common/resolv/herror.c
@@ -48,11 +48,6 @@
  * OF OR IN CONNECTION WITH THE USE OR PERFORMANCE OF THIS SOFTWARE.
  */
 
-#if defined(LIBC_SCCS) && !defined(lint)
-static const char sccsid[] = "@(#)herror.c	8.1 (Berkeley) 6/4/93";
-static const char rcsid[] = "$Id: herror.c,v 1.4 2005/04/27 04:56:41 sra Exp $";
-#endif /* LIBC_SCCS and not lint */
-
 #include "port_before.h"
 
 #include <sys/types.h>
diff --git a/usr/src/lib/libresolv2_joy/common/resolv/res_comp.c b/usr/src/lib/libresolv2_joy/common/resolv/res_comp.c
index c82bf01eb8..abe9b55155 100644
--- a/usr/src/lib/libresolv2_joy/common/resolv/res_comp.c
+++ b/usr/src/lib/libresolv2_joy/common/resolv/res_comp.c
@@ -74,11 +74,6 @@
  * OF OR IN CONNECTION WITH THE USE OR PERFORMANCE OF THIS SOFTWARE.
  */
 
-#if defined(LIBC_SCCS) && !defined(lint)
-static const char sccsid[] = "@(#)res_comp.c	8.1 (Berkeley) 6/4/93";
-static const char rcsid[] = "$Id: res_comp.c,v 1.5 2005/07/28 06:51:50 marka Exp $";
-#endif /* LIBC_SCCS and not lint */
-
 #include "port_before.h"
 #include <sys/types.h>
 #include <sys/param.h>
diff --git a/usr/src/lib/libresolv2_joy/common/resolv/res_data.c b/usr/src/lib/libresolv2_joy/common/resolv/res_data.c
index f1e0dd030b..514a9e56a1 100644
--- a/usr/src/lib/libresolv2_joy/common/resolv/res_data.c
+++ b/usr/src/lib/libresolv2_joy/common/resolv/res_data.c
@@ -20,10 +20,6 @@
  * OF OR IN CONNECTION WITH THE USE OR PERFORMANCE OF THIS SOFTWARE.
  */
 
-#if defined(LIBC_SCCS) && !defined(lint)
-static const char rcsid[] = "$Id: res_data.c,v 1.7 2008/12/11 09:59:00 marka Exp $";
-#endif /* LIBC_SCCS and not lint */
-
 #include "port_before.h"
 
 #include <sys/types.h>
diff --git a/usr/src/lib/libresolv2_joy/common/resolv/res_debug.c b/usr/src/lib/libresolv2_joy/common/resolv/res_debug.c
index e11fb29612..2fcc1ad2ca 100644
--- a/usr/src/lib/libresolv2_joy/common/resolv/res_debug.c
+++ b/usr/src/lib/libresolv2_joy/common/resolv/res_debug.c
@@ -99,11 +99,6 @@
  * IF IBM IS APPRISED OF THE POSSIBILITY OF SUCH DAMAGES.
  */
 
-#if defined(LIBC_SCCS) && !defined(lint)
-static const char sccsid[] = "@(#)res_debug.c	8.1 (Berkeley) 6/4/93";
-static const char rcsid[] = "$Id: res_debug.c,v 1.19 2009/02/26 11:20:20 tbox Exp $";
-#endif /* LIBC_SCCS and not lint */
-
 #include "port_before.h"
 
 #include <sys/types.h>
diff --git a/usr/src/lib/libresolv2_joy/common/resolv/res_findzonecut.c b/usr/src/lib/libresolv2_joy/common/resolv/res_findzonecut.c
index 431c0262c1..c84bd0dec7 100644
--- a/usr/src/lib/libresolv2_joy/common/resolv/res_findzonecut.c
+++ b/usr/src/lib/libresolv2_joy/common/resolv/res_findzonecut.c
@@ -1,7 +1,3 @@
-#if !defined(lint) && !defined(SABER)
-static const char rcsid[] = "$Id: res_findzonecut.c,v 1.10 2005/10/11 00:10:16 marka Exp $";
-#endif /* not lint */
-
 /*
  * Copyright (c) 2004 by Internet Systems Consortium, Inc. ("ISC")
  * Copyright (c) 1999 by Internet Software Consortium.
diff --git a/usr/src/lib/libresolv2_joy/common/resolv/res_init.c b/usr/src/lib/libresolv2_joy/common/resolv/res_init.c
index 10254d1931..4a7724645f 100644
--- a/usr/src/lib/libresolv2_joy/common/resolv/res_init.c
+++ b/usr/src/lib/libresolv2_joy/common/resolv/res_init.c
@@ -73,11 +73,6 @@
  * OF OR IN CONNECTION WITH THE USE OR PERFORMANCE OF THIS SOFTWARE.
  */
 
-#if defined(LIBC_SCCS) && !defined(lint)
-static const char sccsid[] = "@(#)res_init.c	8.1 (Berkeley) 6/7/93";
-static const char rcsid[] = "$Id: res_init.c,v 1.26 2008/12/11 09:59:00 marka Exp $";
-#endif /* LIBC_SCCS and not lint */
-
 #include "port_before.h"
 
 #include <sys/types.h>
diff --git a/usr/src/lib/libresolv2_joy/common/resolv/res_mkquery.c b/usr/src/lib/libresolv2_joy/common/resolv/res_mkquery.c
index cf36855e9d..29cfea2786 100644
--- a/usr/src/lib/libresolv2_joy/common/resolv/res_mkquery.c
+++ b/usr/src/lib/libresolv2_joy/common/resolv/res_mkquery.c
@@ -74,11 +74,6 @@
  * SOFTWARE.
  */
 
-#if defined(LIBC_SCCS) && !defined(lint)
-static const char sccsid[] = "@(#)res_mkquery.c	8.1 (Berkeley) 6/4/93";
-static const char rcsid[] = "$Id: res_mkquery.c,v 1.10 2008/12/11 09:59:00 marka Exp $";
-#endif /* LIBC_SCCS and not lint */
-
 #include "port_before.h"
 #include <sys/types.h>
 #include <sys/param.h>
diff --git a/usr/src/lib/libresolv2_joy/common/resolv/res_mkupdate.c b/usr/src/lib/libresolv2_joy/common/resolv/res_mkupdate.c
index 8f73e281d0..686718b42f 100644
--- a/usr/src/lib/libresolv2_joy/common/resolv/res_mkupdate.c
+++ b/usr/src/lib/libresolv2_joy/common/resolv/res_mkupdate.c
@@ -21,10 +21,6 @@
  * &lt;viraj_bais@ccm.fm.intel.com>
  */
 
-#if !defined(lint) && !defined(SABER)
-static const char rcsid[] = "$Id: res_mkupdate.c,v 1.10 2008/12/11 09:59:00 marka Exp $";
-#endif /* not lint */
-
 #include "port_before.h"
 
 #include <sys/types.h>
diff --git a/usr/src/lib/libresolv2_joy/common/resolv/res_query.c b/usr/src/lib/libresolv2_joy/common/resolv/res_query.c
index 09df3da1fc..222e4d36ef 100644
--- a/usr/src/lib/libresolv2_joy/common/resolv/res_query.c
+++ b/usr/src/lib/libresolv2_joy/common/resolv/res_query.c
@@ -68,11 +68,6 @@
  * SOFTWARE.
  */
 
-#if defined(LIBC_SCCS) && !defined(lint)
-static const char sccsid[] = "@(#)res_query.c	8.1 (Berkeley) 6/4/93";
-static const char rcsid[] = "$Id: res_query.c,v 1.11 2008/11/14 02:36:51 marka Exp $";
-#endif /* LIBC_SCCS and not lint */
-
 #include "port_before.h"
 #include <sys/types.h>
 #include <sys/param.h>
diff --git a/usr/src/lib/libresolv2_joy/common/resolv/res_send.c b/usr/src/lib/libresolv2_joy/common/resolv/res_send.c
index 289db4e77d..764e5c6016 100644
--- a/usr/src/lib/libresolv2_joy/common/resolv/res_send.c
+++ b/usr/src/lib/libresolv2_joy/common/resolv/res_send.c
@@ -74,11 +74,6 @@
  * SOFTWARE.
  */
 
-#if defined(LIBC_SCCS) && !defined(lint)
-static const char sccsid[] = "@(#)res_send.c	8.1 (Berkeley) 6/4/93";
-static const char rcsid[] = "$Id: res_send.c,v 1.22 2009/01/22 23:49:23 tbox Exp $";
-#endif /* LIBC_SCCS and not lint */
-
 /*! \file
  * \brief
  * Send query to name server and wait for reply.
diff --git a/usr/src/lib/libresolv2_joy/common/resolv/res_update.c b/usr/src/lib/libresolv2_joy/common/resolv/res_update.c
index df24aee3bd..bf95aef48d 100644
--- a/usr/src/lib/libresolv2_joy/common/resolv/res_update.c
+++ b/usr/src/lib/libresolv2_joy/common/resolv/res_update.c
@@ -1,7 +1,3 @@
-#if !defined(lint) && !defined(SABER)
-static const char rcsid[] = "$Id: res_update.c,v 1.13 2005/04/27 04:56:43 sra Exp $";
-#endif /* not lint */
-
 /*
  * Copyright (c) 2004 by Internet Systems Consortium, Inc. ("ISC")
  * Copyright (c) 1996-1999 by Internet Software Consortium.
